js/server/modules/@arangodb/foxx/manager.js

```diff
@@ -314,9 +315,13 @@ function startup () {
 function upsertSystemServices () {
   const serviceDefinitions = new Map();
   for (const mount of SYSTEM_SERVICE_MOUNTS) {
-    const serviceDefinition = utils.getServiceDefinition(mount) || {mount};
-    const service = FoxxService.create(serviceDefinition);
-    serviceDefinitions.set(mount, service.toJSON());
+    try {
+      const serviceDefinition = utils.getServiceDefinition(mount) || {mount};
+      const service = FoxxService.create(serviceDefinition);
+      serviceDefinitions.set(mount, service.toJSON());
+    } catch (e) {
+      console.errorStack(e);
```

Comment:
`upsertSystemServices` should not blow up because a single system service is broken, but we still want to log any errors because they may indicate a serious problem.